### PR TITLE
Keeper script create jobs

### PR DESCRIPTION
**Pull request by cmalec**

### CODE REVIEW

File: core/scripts/chaincli/handler/keeper_launch.go [hunk 1]
```diff
@@ -126,9 +126,9 @@ func (k *Keeper) LaunchAndTest(ctx context.Context, withdraw bool) {
 		}
 
 		// Create authenticated client
-		remoteNodeURL, err := url.Parse(startedNode.url)
-		if err != nil {
-			log.Fatal(err)
+		remoteNodeURL, parseErr := url.Parse(startedNode.url)
+		if parseErr != nil {
+			log.Fatal(parseErr)
```
Comment (begmaroman) [inline core/scripts/chaincli/handler/keeper_launch.go:131]: could you share a reason for this change?

**Review by begmaroman — APPROVED** [review]
LGTM, just minor question

**Comment by cmalec** [comment]
@FelixFan1992

**Review by FelixFan1992 — DISMISSED** [review]

**Comment by infiloop2** [comment]
Removing minIncomingConfirmation from job spec as we should use the default per chain: https://github.com/smartcontractkit/chainlink/blob/develop/core/services/keeper/delegate.go#L79-L82

**Pull request merged**